profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_community/tests/multisite_business_indicators_community.test

```diff
@@ -129,30 +130,32 @@ class BusinessIndicatorsCommunityQAWebTestCase extends DrupalWebTestCase {
           $this->assertTrue((bool) $results, "View '{$view_name}' display '{$display_id}' not empty");
           $view->set_display($display_id);
           $title = $view->display[$display_id]->handler->get_option('title');
-          $this->assertText($title, "Block '{$title}' found on overview page.");          
+          $this->assertText($title, "Block '{$title}' found on overview page.");
         }
       }
     }
-
-    return;
   }
 
-  
-  function drupalCreateComment($node, $comment = array()) {
-     $comment += array(
+
+  /**
+   * Test comment creation.
+   */
```

Comment:
Verbs in docblocks should be in the passive sense, so "Tests comment creation.".